Sources/MongoSwift/Operations/RenameCollectionOperation.swift

```diff
@@ -0,0 +1,72 @@
+import CLibMongoC
+
+/// Options to use when renaming a collection. These options will be used for `MongoCollection.renamed`.
+public struct RenameCollectionOptions: Codable {
+    /// Specifies whether an existing collection matching the new name should be dropped before the rename.
+    /// If this is not set to true and a collection with the new collection name exists, the server will throw an error.
+    public let dropTarget: Bool?
```

Comment:
this should be a `var`, not a `let`, so users can mutate this property on an existing instance of `RenameCollectionOptions`.